modules/data-loading/pages/kafka-loader/index.adoc

```diff
@@ -20,7 +20,6 @@ If the loader is interrupted, you have to resume the loading job manually by run
 This resumes the loading job where it was interrupted.
 
 == Supported file formats
-* Avro
```

Comment:
Keep Avro, but add a note with a hyperlink that says something like
"Avro (when sourced from an external Kafka cluster using MirrorMaker)"